src/index.css

```diff
@@ -4,6 +4,7 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap');
 @import 'simplebar/src/simplebar.css';
 @import 'react-datepicker/dist/react-datepicker.css';
+@import 'swiper/swiper.min.css';
```

Comment:
Please move import to the file where it is needed

Reply:
It has been updated.